windows_docker/datadog_checks/windows_docker/data/conf.yaml.example

```diff
@@ -0,0 +1,126 @@
+init_config:
```

Comment:
Hello :) In order for the CI tests to pass, you must follow the configuration file norm described here: https://docs.datadoghq.com/developers/integrations/new_check_howto/#configuration-file

Reply:
hello :) 

So Every commented line should at least have two `#` in front of it.
Each parameter should be preceded with a  `@param <name> - <type> - optional - default: <defval>` line

You may find an example of the expected configuration file shape here: https://docs.datadoghq.com/developers/integrations/new_check_howto/#example-configuration 

Feel free to get back to me if you have any questions :)

Reply:
`7875b2a` attempts to fix the config file formatting.